build/buildTenDRA.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -ex
+
+VERSION=$1
+if echo ${VERSION} | grep 'trunk'; then
+    VERSION=trunk-$(date +%Y%m%d)
+    BRANCH=main
+else
+    BRANCH=V${VERSION}
+fi
+
+OUTPUT=/root/tendra-${VERSION}.tar.xz
+S3OUTPUT=""
+if echo $2 | grep s3://; then
+    S3OUTPUT=$2
+else
+    OUTPUT=${2-/root/tendra-${VERSION}.tar.xz}
+fi
+
+PREFIX=$(pwd)/prefix
+DIR=$(pwd)/tendra
+git clone --depth 1 -b ${BRANCH} https://github.com/tendra/tendra.git ${DIR}
+
+# no -j (currently breaks)
+pmake -C ${DIR} TARGETARCH=x32_64
```

Comment:
Hmm, I was looking at if this could not "just" be x86 or i386, but it seems in some places only this value is accepted in this situation :(